lib/apis/recommend_api.dart

```diff
@@ -14,6 +14,45 @@ class RecommendAPI {
   static const String _articles = '$_api/article';
   static const String _pins = '$_api/short_msg';
 
+  static Future<ResponseModel<FeedModel>> getArticles({
+    bool isFollow = false,
+    String? lastId,
+    String? cateId,
+    String? tagId,
+    int sortType = 200,
+    int limit = 20,
+    int idType = 2,
```

Comment:
`idType` doesn't indicate valid things. Make it self-describe or add comments for it.